pkg/reconciler/openshift/tektonresult/extension.go

```diff
@@ -141,3 +150,79 @@ func filterAndTransform() client.FilterAndTransform {
 		return manifest, nil
 	}
 }
+
+// injectBoundSAToken adds a sa token projected volume to the Results Deployment
+func injectBoundSAToken(comp v1alpha1.TektonComponent) mf.Transformer {
```

Comment:
can we add it in the doc about this injection of SA ?

Reply:
It's being mentioned in release note. For now it's enough. We planned to update install section of Results. We can add then.

Reply:
TBH, it doesn't change have any UX change for end user, that's why I didn't add it.